paper/content/well-typed-holes.tex

```diff
@@ -16,14 +16,13 @@ \subsection{Well-Typed Holes}
 Barebones language support should at least include the ability to inspect the
 type of a hole and the local context it appears in, as well as the ability to
 safely evaluate programs that still contain holes.
-\todo{Implement \texttt{instantiate}? Shouldn't take too long.}
```

Comment:
The paragraph needs to be edited to add hole instantiation as one of the barebones features.
I remarked yesterday that we were suspiciously only describing what we had already implemented
when, for proper "interactive" programming you do need to be able to fill holes, hence the feature.

Reply:
okay! Apologies for the premature deletion of the 'todo'.